Update hdinsight-restrict-public-connectivity.md (#111105)
Azure/azure-content: articles/hdinsight/hdinsight-restrict-public-connectivity.md

- SQL metastores: Apache Ranger, Ambari, Oozie, and Hive
- Azure Key Vault 

> It isn't mandatory to use private endpoints for these resources. But if you plan to use private endpoints for these resources, you must create the resources and configure the private endpoints and DNS entries before you create the HDInsight cluster. All these resources should be accessible from inside the cluster subnet, either through a private endpoint or otherwise. If you are planning to use a private endpoint its recommended to leverage the cluster subnet.

Reviewer: ```suggestion
It isn't mandatory to use private endpoints for these resources. But if you plan to use private endpoints for these resources, you must create the resources and configure the private endpoints and DNS entries before you create the HDInsight cluster. All these resources should be accessible from inside the cluster subnet, either through a private endpoint or otherwise. If you're planning to use a private endpoint, it's recommended to leverage the cluster subnet.
```
